geode-docs/managing/cache_snapshots/exporting_a_snapshot.html.md.erb

```diff
@@ -60,15 +60,37 @@ region.getSnapshotService().save(mySnapshot, SnapshotFormat.GEMFIRE);
 Open a gfsh prompt. After connecting to a <%=vars.product_name%> distributed system, at the prompt type:
 
 ``` pre
-gfsh>export data --region=Region --file=filename.gfd 
---member=membername
+gfsh>export data --region=Region --file=FileName.gfd --member=MemberName
 ```
 
-where *Region* corresponds to the name of the region that you want to export, *filename* (must end in .gfd) corresponds to the name of the export file and *membername* corresponds to a member where the region to export is hosted. For example:
+where *Region* corresponds to the name of the region that you want to export, *FileName* (must end in .gfd) corresponds to the name of the export file and *MemberName* corresponds to a member where the region to export is hosted. For example:
 
 ``` pre
-gfsh>export data --region=region1 --file=region1_2012_10_10.gfd 
---member=server1
+gfsh>export data --region=region1 --file=region1_2012_10_10.gfd --member=server1
 ```
 
 The snapshot file will be written on the remote member at the location specified by the `--file` argument. For example, in the example command above, the `region1_2012_10_10.gfd` file will be written in the working directory of `server1`. For more information on this command, see [export data](../../tools_modules/gfsh/command-pages/export.html#topic_263B70069BFC4A7185F86B3272011734).
+
+###<a id="export_example_with_options" class="no-quick-link"></a>Export Example with Options
+
+These examples show how to include the `parallel` option for exporting partitioned regions. 
+Note that the `parallel` option takes a directory rather than a file; see [export data](../../tools_modules/gfsh/command-pages/export.html#topic_263B70069BFC4A7185F86B3272011734) for details.
+
+**Java API:**
+
+``` pre
+File mySnapshotDir = ...
+Region<String, MyObject> region = ... 
+
+SnapshotOptions<Integer, MyObject> options = region.getSnapshotServive.createOptions().setParallelMode(true);
```

Comment:
Please inject newlines into the code fragment such that the output will not have a horizontal scroll bar.